src/purify.ts

```diff
@@ -1840,12 +1857,15 @@ interface DOMPurify {
   /**
    * Removes all DOMPurify hooks at a given entryPoint
    *
-   * @param entryPoint entry point for the hooks to remove
+   * @param {HookName} entryPoint entry point for the hooks to remove
+   * @returns {void}
    */
   removeHooks(entryPoint: HookName): void;
 
   /**
    * Removes all DOMPurify hooks.
+   *
+   * @returns {void}
```

Comment:
This JSDoc comment should be redundant since the return type annotation indicates the method returns void. @cure53 , do you want all functions to have an `@returns` JSDoc tag, even if they don't return a value?